op-node/rollup/chain_spec.go

```diff
@@ -33,17 +33,18 @@ const maxSequencerDriftFjord = 1800
 type ForkName string
 
 const (
-	Bedrock  ForkName = "bedrock"
-	Regolith ForkName = "regolith"
-	Canyon   ForkName = "canyon"
-	Delta    ForkName = "delta"
-	Ecotone  ForkName = "ecotone"
-	Fjord    ForkName = "fjord"
-	Granite  ForkName = "granite"
-	Holocene ForkName = "holocene"
-	Isthmus  ForkName = "isthmus"
-	Jovian   ForkName = "jovian"
-	Interop  ForkName = "interop"
+	Bedrock        ForkName = "bedrock"
+	Regolith       ForkName = "regolith"
+	Canyon         ForkName = "canyon"
+	Delta          ForkName = "delta"
+	Ecotone        ForkName = "ecotone"
+	Fjord          ForkName = "fjord"
+	Granite        ForkName = "granite"
+	Holocene       ForkName = "holocene"
+	Isthmus        ForkName = "isthmus"
+	Jovian         ForkName = "jovian"
+	Interop        ForkName = "interop"
+	CustomGasToken ForkName = "custom_gas_token"
```

Comment:
Not exactly sure how to feature flag the best way, this cgt fork should be temporary